cnv/cnv_virtual_machines/cnv_vm_networking/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -16,6 +16,14 @@ include::modules/cnv-networking-glossary.adoc[leveloffset=+1]
 
 == Creating a NetworkAttachmentDefinition
 
+.Prerequisites
+
+* A Linux bridge must be xref:../../../cnv/cnv_node_network/cnv-updating-node-network.adoc[configured and attached] to the correct
+Network Interface Card (NIC) on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
+* The NIC must be tagged to all relevant VLANs.
+** For example: `bridge vlan add dev bond0 vid 1-4095 master`
```

Comment:
We can drop this and just reference the bridge configuration in nmstate article

Reply:
@phoracek The 'For Example' line, or the 'The NIC must be tagged...' line as well?

Reply:
Mmm. Depends. Do we want to simply tell users "bridge is needed, see NMSTATE in order to learn how to create one" or do we want to say "bridge with vlan filtering X and Y is needed, DIY or use NMSTATE". I would vote for the former. I want all the souls.

Reply:
Please either keep only the first bullet point OR split it in two paragraphs, first with recommended configuration via nmstate, second as an alternative with the original prerequisites.

Reply:
rewrote the first:
"* A Linux bridge must be configured and attached on every node. 
See the xref:../../../cnv/cnv_node_network/cnv-updating-node-network.adoc[node networking] section for more information."
I also prefer the more definitive approach